### PR TITLE
sunpy.util python file review

### PR DESCRIPTION
**Pull request by nabobalis**
This time `util`.

Why does the bot hate me?

Also I am not sure if I broke anything, the CI will tell me.

### CODE REVIEW

File: sunpy/data/_sample.py [hunk 1]
```diff
@@ -127,7 +126,7 @@ def get_sample_file(filename, url_list, show_progress=True, overwrite=False,
                 online_filename += '?raw=true'
             try:
                 url = urljoin(base_url, online_filename)
-                exists = url_exists(url)
+                exists = True
```
Comment (nabobalis) [inline sunpy/data/_sample.py]: This is not used in the sample data PR #2993

File: sunpy/util/__init__.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
-"""SunPy utility functions"""
```
Comment (nabobalis) [inline sunpy/util/__init__.py]: We currently don't import all of the util modules into this name space.

**Comment by Cadair** [comment]
Can you update this @nabobalis ?

File: sunpy/data/_sample.py (hunk 1 above)
Comment (Cadair) [reply sunpy/data/_sample.py]: This is now merged.

**Comment by Cadair** [comment]
I am all in favour of this but we are going to need to pull the coditional dispatch code into radiospectra before 1.0

**Comment by nabobalis** [comment]
That stuff is now in radiospectra. I added it when I redid the CI. I will also update it tonight.

**Pull request merged**